Tools/GNUMake/Make.machines

```diff
@@ -34,6 +34,10 @@ ifeq ($(NERSC_HOST), alvarez)
   which_site := nersc
   which_computer := perlmutter
 endif
+ifeq ($(NERSC_HOST), muller)
+  which_site := nersc
+  which_computer := perlmutter
```

Comment:
```suggestion
  which_computer := muller
```
?

Reply:
Or is it the same as perlmutter?

Reply:
Muller is like Alvarez, a test machine that should be treated like Perlmutter. So, `which_computer=perlmutter` gives the same configuration without any additional code changes in sites/Make.nersc.

Also, the future of Muller and Alvarez may be to merge them into Perlmutter? So, I don't think anything fancy needs to be done with either of them.

Reply:
I figured that might be the case.  Anyway, could we add a comment?

Reply:
Sure! How's that one look?